genie-web/src/main/resources/genie-web-defaults.yml

```diff
@@ -141,7 +141,7 @@ spring:
       repositories:
         enabled: false
   datasource:
-    url: jdbc:h2:mem:genie
+    url: jdbc:h2:mem:genie;MODE=MySQL;DB_CLOSE_DELAY=-1;DATABASE_TO_LOWER=TRUE;CASE_INSENSITIVE_IDENTIFIERS=TRUE;NON_KEYWORDS=value,limit;
```

Comment:
I assume our test case is comprehensive, and since all test cases are passed, I think it should be safe but want to bring this up again to double confirm that there should have no risk with such change.

Reply:
Yes. It is safe. This change is required as H2 is upgraded from 1.4 to 2.1.